test/e2e/charts/flow-visibility/templates/namespace.yaml

```diff
@@ -0,0 +1,8 @@
+{{- if .Values.secureConnection.enable }}
```

Comment:
this conditional doesn't seem right to me.
why do we only define the Namespace when TLS is enabled?

Reply:
After thinking about this some more, it seems to me because the only resource we create in the `flow-aggregator` namespace is the `clickhouse-ca` secret. However it is weird that this chart creates the flow-aggregator Namespace, or that it creates any resource in the flow-aggregator Namespace at all for that matter.

IMO, a better solution would be to create the secret in the `flow-visibility` namespace, then have the test framework copy the secret with the CA to the `flow-aggregator` namespace, so it can be accessed by the FA. This is actually a pretty common solution AFAIK.

Reply:
I was doing this because the flow-visibility.yaml, which includes the `clickhouse-ca`, is applied before the flow-aggregator.yaml in the e2e test. I also think it's weird to put resources with different namespaces together.

> IMO, a better solution would be to create the secret in the flow-visibility namespace, then have the test framework

It makes sense to me. I've deleted the namespace.yaml and copied the secret in the framework. Thanks!.